Cutelyst/utils.cpp

```diff
@@ -287,3 +287,133 @@ QString Utils::decodePercentEncoding(QByteArray *ba)
         return QString::fromUtf8(ba->data(), outlen);
     }
 }
+
+std::chrono::microseconds Utils::durationFromString(const QString &str, bool *ok)
```

Comment:
I think this should take a QStringView and operate on it (ie no toLatin1 which allocates), or operate over QByteArrayView